### PR TITLE
Simplify experience when additional scopes are required to view the d…

### PR DESCRIPTION
**Pull request by sashadoes**
## Summary

Addresses issue:

- #5497 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: assets/js/components/PermissionsModal/AuthenticatedPermissionsModal.js [hunk 1]
```diff
@@ -93,6 +96,15 @@ const AuthenticatedPermissionsModal = () => {
 		return null;
 	}
 
+	if (
+		unsatisfiedScopes &&
+		permissionsError?.data?.scopes.every( ( i ) =>
+			unsatisfiedScopes.includes( i )
```
Comment (hussain-t) [inline assets/js/components/PermissionsModal/AuthenticatedPermissionsModal.js:102]: Can we rename it to `scope` for clarity?

```suggestion
		permissionsError?.data?.scopes.every( ( scope ) =>
			unsatisfiedScopes.includes( scope )
```

**Review by hussain-t — CHANGES_REQUESTED** [review]
Thanks, @sashadoes. Overall the changes LGTM. However, I’ve left a little comment to rename a variable. Also, it would be nice to add a test case to verify the modal doesn’t appear in `assets/js/components/PermissionsModal/index.test.js`. WDYT?

**Review by hussain-t — APPROVED** [review]
Thanks, @sashadoes. LGTM 👍

**Review by eugene-manuilov — APPROVED** [review]
LGTM

**Pull request merged**